docs/src/modules/ROOT/pages/optimization-algorithms/overview.adoc

```diff
@@ -637,6 +637,126 @@ This is useful for benchmarking.
 
 Switching xref:using-timefold-solver/running-the-solver.adoc#environmentMode[EnvironmentMode] can heavily impact when this termination ends.
 
+[#diminishedReturnsTermination]
+==== `DiminishedReturnsTermination`
```

Comment:
I'd start with an introductory paragraph explaining why we've done this. Absolute terminations are hard to configure, because they either depend on hardware (time-based) or on the problem (score-based). The relative termination doesn't have this problem.